src/main/java/seedu/duke/Parser.java

```diff
@@ -47,6 +47,23 @@ static void setProjectIndex(int newIndex) {
         projectIndex = newIndex;
     }
 
+    public static int extractIndex(String [] inputs) {
```

Comment:
In future, when we assign members to project, we might need to call extractIndex twice and extract memberIndex and projectIndex. Consider it to take in a input "word" instead of the whole input array?